File: Actividad 5/Problema 1/problema1_cs/FacturaTest.cs
using System;

class FacturaTest
{
    public static void Main(string[] args)
    {
        Factura ferre = new Factura();

        Console.WriteLine("Ingresa el numero de pieza: ");
        ferre.numeroPieza = Console.ReadLine();

        Console.WriteLine("Ingresa la descripcion: ");
        ferre.descripcion = Console.ReadLine();

        Console.WriteLine("Ingresa la cantidad: ");
        ferre.cantidad = int.Parse(Console.ReadLine());

        Console.WriteLine("Ingrese el precio: ");
        ferre.precioArticulo = double.Parse(Console.ReadLine());

        if (ferre.precioArticulo < 0)
            ferre.precioArticulo = 0.0;

        if (ferre.cantidad < 0)
            ferre.cantidad = 0;

        Console.WriteLine("Factura: " + ferre.obtenerMontoFactura());
    }
}

File: Actividad 5/Problema 1/problema1_java/FacturaTest.java
import java.util.Scanner;

public class FacturaTest 
{
    public static void main(String args[])
    {
        Factura ferre = new Factura();
        Scanner input = new Scanner(System.in);
        
        System.out.println("Ingresa el numero de pieza: ");
        ferre.numeroPieza = input.nextLine();
        
        System.out.println("Ingresa la descripcion: ");
        ferre.descripcion = input.nextLine();
        
        System.out.println("Ingresa la cantidad: ");
        ferre.cantidad = input.nextInt();
        
        System.out.println("Ingrese el precio: ");
        ferre.precioArticulo = input.nextDouble();
        
        if (ferre.precioArticulo < 0) ferre.precioArticulo = 0.0;
        if (ferre.cantidad < 0) ferre.cantidad = 0;
        
        System.out.println("Factura: " + ferre.obtenerMontoFactura());
        
    }
}